src/Compression/Compression.cs

```diff
@@ -322,14 +322,19 @@ public static DeflateStreamNative Create (Stream compressedStream, CompressionMo
 			Dispose (false);
 		}
 
+		public void Dispose()
+		{ 
+			Dispose(true);
+			GC.SuppressFinalize(this);
+		}
```

Comment:
minor (in case you update the PR again): you're missing spaces before the `(`

Reply:
@spouliot lets do it :)